### DISCUSSION TITLE
How can I set editable resource targets by role?

### DISCUSSION THREAD

**Discussion by mukitaro** [gravitational/teleport#15598] [Q&A] [answered]
Hello,

I have a question about role permission settings.

For example, We have Role-A and Role-B.
Is it possible to set User-A to be able to edit only Role-A?

Thanks!

**Comment by stevenGravy** [comment, marked as answer]
Currently no. There is not a `where` setting for `roles` currently.

**Reply by mukitaro** [reply]
Hi stevenGravy

Thank you for providing an answer to my question.
I understood it.
